src/control/server/mgmt_check.go

```diff
@@ -283,6 +277,23 @@ func (svc *mgmtSvc) SystemCheckStart(ctx context.Context, req *mgmtpb.CheckStart
 	return resp, nil
 }
 
+func (svc *mgmtSvc) mergePoliciesWithCurrent(policies []*mgmtpb.CheckInconsistPolicy) ([]*mgmtpb.CheckInconsistPolicy, error) {
+	pm, err := svc.getCheckerPolicyMap()
+	if err != nil {
+		return nil, err
+	}
+
+	// Allow the requested policies to override any policies stored in the policy map.
+	for _, pol := range policies {
+		pm[pol.InconsistCas] = pol
+	}
+	result := make([]*mgmtpb.CheckInconsistPolicy, 0, len(pm))
+	for _, pol := range pm {
```

Comment:
Rather than looping through, why not use copy() on pm.ToSlice()?

Reply:
Ah, we could actually just use `pm.ToSlice()` without a copy. That's a good idea. I'll change it if there are any other change requests, if that's okay with you.